### DISCUSSION TITLE
Video Upload Error

### DISCUSSION THREAD

**Discussion by iali9906** [outline/outline#7030] [Self Hosting] [answered]
HI,
i can't upload an embedded videos, i see this error in docker logs:

outline-prod-01        | {"event":{"actorId":"ca39d6ba-e8cf-41c6-a43a-b30fd0f6f244","changes":null,"collectionId":"ee5ba633-1ff6-489f-a62c-d34ca9a3ace4","createdAt":"2024-06-13T07:06:09.952Z","data":{"title":"Procedure"},"documentId":"848a73cb-e176-410a-a0ac-1385fd18dc6a","id":"909c8f42-b5f0-4023-93da-edeefa0b2ace","ip":"91.224.72.1","modelId":"fff69a2e-f003-4078-bd35-c53a4af36d93","name":"views.create","teamId":"3a2bb618-a815-4a39-bd74-d3befb6e378e","userId":null},"label":"worker","level":"info","message":"WebhookProcessor running views.create"}
outline-prod-01        | Error: Parse Error
outline-prod-01        |     at socketOnEnd (node:_http_server:800:22)
outline-prod-01        |     at TLSSocket.emit (node:events:530:35)
outline-prod-01        |     at TLSSocket.emit (node:domain:488:12)
outline-prod-01        |     at endReadableNT (node:internal/streams/readable:1696:12)
outline-prod-01        |     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
outline-prod-01        |   bytesParsed: 0,
outline-prod-01        |   code: 'HPE_INVALID_EOF_STATE',
outline-prod-01        |   reason: 'Invalid EOF state'
outline-prod-01        | }
outline-prod-01        | FormidableError: Request aborted
outline-prod-01        |     at IncomingMessage.<anonymous> (/opt/outline/node_modules/formidable/src/Formidable.js:197:21)
outline-prod-01        |     at IncomingMessage.emit (node:events:518:28)
outline-prod-01        |     at IncomingMessage.emit (node:domain:488:12)
outline-prod-01        |     at IncomingMessage._destroy (node:_http_incoming:224:10)
outline-prod-01        |     at _destroy (node:internal/streams/destroy:121:10)
outline-prod-01        |     at IncomingMessage.destroy (node:internal/streams/destroy:83:5)
outline-prod-01        |     at abortIncoming (node:_http_server:794:9)
outline-prod-01        |     at socketOnClose (node:_http_server:788:3)
outline-prod-01        |     at TLSSocket.emit (node:events:530:35)
outline-prod-01        |     at TLSSocket.emit (node:domain:488:12)
outline-prod-01        |     at node:net:337:12
outline-prod-01        |     at Socket.done (node:_tls_wrap:657:7)
outline-prod-01        |     at Object.onceWrapper (node:events:633:26)
outline-prod-01        |     at Socket.emit (node:events:530:35)
outline-prod-01        |     at Socket.emit (node:domain:488:12)
outline-prod-01        |     at TCP.<anonymous> (node:net:337:12) {
outline-prod-01        |   code: 1002,
outline-prod-01        |   httpCode: 500
outline-prod-01        | }

outline-prod-01        | {"event":{"actorId":"6361a5fb-4e4d-483a-a540-09024840345b","collectionId":"6b207735-2d69-4e5e-8173-f1cb3c1151b4","createdAt":"2024-06-13T07:00:18.928Z","data":{"done":false,"multiplayer":true,"title":"Service Desk"},"documentId":"d08819a3-8fbc-49f8-8ebb-92d580c174bc","id":"3d62c321-881d-4cd9-a841-951f57daa34c","name":"documents.update.delayed","teamId":"3a2bb618-a815-4a39-bd74-d3befb6e378e"},"label":"worker","level":"info","message":"WebhookProcessor running documents.update.delayed"}
outline-prod-01        | Error: Parse Error
outline-prod-01        |     at socketOnEnd (node:_http_server:800:22)
outline-prod-01        |     at TLSSocket.emit (node:events:530:35)
outline-prod-01        |     at TLSSocket.emit (node:domain:488:12)
outline-prod-01        |     at endReadableNT (node:internal/streams/readable:1696:12)
outline-prod-01        |     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
outline-prod-01        |   bytesParsed: 0,
outline-prod-01        |   code: 'HPE_INVALID_EOF_STATE',
outline-prod-01        |   reason: 'Invalid EOF state'
outline-prod-01        | }
outline-prod-01        | FormidableError: Request aborted
outline-prod-01        |     at IncomingMessage.<anonymous> (/opt/outline/node_modules/formidable/src/Formidable.js:197:21)
outline-prod-01        |     at IncomingMessage.emit (node:events:518:28)
outline-prod-01        |     at IncomingMessage.emit (node:domain:488:12)
outline-prod-01        |     at IncomingMessage._destroy (node:_http_incoming:224:10)
outline-prod-01        |     at _destroy (node:internal/streams/destroy:121:10)
outline-prod-01        |     at IncomingMessage.destroy (node:internal/streams/destroy:83:5)
outline-prod-01        |     at abortIncoming (node:_http_server:794:9)
outline-prod-01        |     at socketOnClose (node:_http_server:788:3)
outline-prod-01        |     at TLSSocket.emit (node:events:530:35)
outline-prod-01        |     at TLSSocket.emit (node:domain:488:12)
outline-prod-01        |     at node:net:337:12
outline-prod-01        |     at Socket.done (node:_tls_wrap:657:7)
outline-prod-01        |     at Object.onceWrapper (node:events:633:26)
outline-prod-01        |     at Socket.emit (node:events:530:35)
outline-prod-01        |     at Socket.emit (node:domain:488:12)
outline-prod-01        |     at TCP.<anonymous> (node:net:337:12) {
outline-prod-01        |   code: 1002,
outline-prod-01        |   httpCode: 500
outline-prod-01        | }

thx

**Comment by tommoor** [comment, marked as answer]
Unclear other than this being related to an unexpected end of the HTTP request. Most likely scenario is that you have some proxy or gateway infront of the server that is restricting request size, which would be why it only shows up with larger files.
